app/src/main/res/menu/menu_garden_action_mode.xml

```diff
@@ -15,6 +14,12 @@
   ~ limitations under the License.
   -->
 
-<menu xmlns:android="http://schemas.android.com/apk/res/android">
-    <!-- TODO: Add ActionMode actions. -->
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
```

Comment:
PTAL. 
NO. empty line between Line 18 - Line 19?

Reply:
+1 for newline to match other menu XML files

Reply:
Done. :)